src/eventshub/src/component/Search/Search.tsx

```diff
@@ -65,15 +65,16 @@ const Search = () => {
                 direction='col'
                 className='er-hidden md:er-block er-col-span-12 md:er-col-span-4 lg:er-col-span-3'
               >
-                <Autocomplete
-                  searchClient={algoliaClient}
-                  searchIndex={indexName}
-                  placeholder={placeholder}
-                  detachedMediaQuery='none'
-                  openOnFocus
-                  activeTab={activeTab}
-                  activeField={activeField}
-                />
+                <div className="er-mb-30 md:er-mb-36">
```

Comment:
Moved this div here. It was inside the autocomplete component before and could go back in if the mobile stuff is janky.